packages/autocomplete-core/src/getPropGetters.ts

```diff
@@ -55,6 +55,18 @@ export function getPropGetters<
 
         if (isTargetWithinAutocomplete === false) {
           store.dispatch('blur', null);
+
+          if (!props.debug && onInput.isRunning()) {
```

Comment:
Although it's would be duplicated, it's worth adding a comment to that branch (and other similars) to explain why it's required. It's quite confusing without context.

Reply:
Addressed in b47de3f.